lib/debug/config.rb

```diff
@@ -84,6 +84,11 @@ def self.parse_argv argv
       o.separator ''
 
       o.separator 'Debug console mode:'
+      o.on("-h", "--help", "Print help") do
```

Comment:
It's not console mode help, so it should be located at the end with some separator.